src/declarations/stencil-private.ts

```diff
@@ -1334,6 +1334,19 @@ export interface RenderNode extends HostElement {
    */
   ['s-hn']?: string;
 
+  /**
+   * Slot host tag name:
+   * This is the tag name of the element where this node
+   * has been moved to during slot relocation.
+   *
+   * This allows us to check if the node has been moved and prevent
+   * us from thinking a node _should_ be moved when it may already be in
+   * its final destination.
+   *
+   * This value is reset whenever the node is put back into its original location.
```

Comment:
Minor, can we clarify what 'reset' means in this context? (IE 'what value does it get when reset?')